dist/index.js

```diff
@@ -8274,110 +8274,108 @@ function wrappy (fn, cb) {
 /***/ }),
 
 /***/ 8919:
-/***/ (function(__unused_webpack_module, exports, __nccwpck_require__) {
+/***/ ((__unused_webpack_module, exports, __nccwpck_require__) => {
 
 "use strict";
 
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
```

Comment:
I hope this isn't the actual codebase all in a single file. If this is a build output, I'm not sure this should be committed to the git repo at all without good reason.

Reply:
The distributed file must be in the repo to have the action work.